### PR TITLE
Add a pull request for Hug-YQ-Hopkinton

### LINKED ISSUE #40: Add a file for hug-yq hopkinton
# Adding file for hug-yq hopkinton

-[]create branch
-[]add file
-[]commit change
-[]open request
-[]get code review

### PR DESCRIPTION
**Pull request by hug-yq**
This pull request resolves #40 
@ntran can you see this?

### CODE REVIEW

File: hug-yq-hopkinton.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Hopkinton,MA
```
Comment (goodwb1) [inline hug-yq-hopkinton.md]: Are topics needed?

Comment (goodwb1) [inline hug-yq-hopkinton.md]: comment for review.

Comment (hug-yq) [inline hug-yq-hopkinton.md]: guess so :)

Comment (hug-yq) [inline hug-yq-hopkinton.md]: sure

**Pull request merged**